### PR TITLE
add signal for event bulk actions

### PR DESCRIPTION
**Pull request by jeriox**

### CODE REVIEW

File: ephios/core/templatetags/event_extras.py [hunk 1]
```diff
@@ -77,3 +80,14 @@ def event_signup_state_counts(event, user):
                 counter[participation.state] += 1
                 break
     return counter
+
+
+@register.simple_tag(name="event_bulk_actions")
+def event_bulk_actions():
+    return format_html(
+        reduce(
+            lambda result, item: result + "".join(item[1]),
```
Comment (felixrindt) [inline ephios/core/templatetags/event_extras.py:89]: Can you use nested tuple unpacking? 
```suggestion
            lambda html, (receiver, buttons): html + "".join(buttons),
```

**Review by felixrindt — APPROVED** [review]

File: ephios/core/templatetags/event_extras.py (hunk 1 above)
Comment (jeriox) [reply ephios/core/templatetags/event_extras.py:89]: tuple parameter unpacking is not supported in python3

Comment (felixrindt) [reply ephios/core/templatetags/event_extras.py]: Oh okay. Do you see another way to make that more readable?

Comment (jeriox) [reply ephios/core/templatetags/event_extras.py]: is this comment based on the new code?

Comment (felixrindt) [reply ephios/core/templatetags/event_extras.py]: Probably not

**Review by felixrindt — APPROVED** [review]

File: ephios/core/signals.py [hunk 2]
```diff
@@ -64,6 +64,14 @@
 This signal is called periodically, at least every 15 minutes.
 """
 
+register_event_bulk_action = PluginSignal()
+"""
+This signal is sent out to get a list of actions that a user can perform on a list of events.
+Receivers should return a list of actions. Each action is represented by a dict with the keys ``url``, ``title`` and ``icon``.
```
Comment (felixrindt) [inline ephios/core/signals.py:70]: `title` should be `label` maybe?

**Pull request merged**